x-pack/plugins/security_solution/public/app/solution_navigation/links/sections/ml_links.ts

```diff
@@ -39,7 +39,7 @@ export const mlAppLink: LinkItem = {
   id: SecurityPageName.mlLanding,
   title: i18n.ML_TITLE,
   path: MACHINE_LEARNING_PATH,
-  capabilities: [`${SERVER_APP_ID}.show`],
+  capabilities: [[`${SERVER_APP_ID}.show`, `ml.canGetMlInfo`]],
```

Comment:
@jgowdyelastic Okay, that makes sense. We picked the `canGetMlInfo` capability quite blindly because it looked like a "_read_" access capability. 
Thanks for the clarification, we will update the code accordingly. 
And feel free to change them on your own if the criteria for those capabilities change 👍